runtime/compiler/z/codegen/J9TreeEvaluator.cpp

```diff
@@ -2244,8 +2244,16 @@ J9::Z::TreeEvaluator::inlineMathFma(TR::Node *node, TR::CodeGenerator *cg)
    TR::Register      * v2      = cg->evaluate(node->getSecondChild());
    TR::Register      * v3      = cg->evaluate(node->getThirdChild());
```

Comment:
We can get a evaluated third child in a register that can be clobbered and can use that as targetRegister. This will save you an additional instruction you need to generate to load third operand to a target register.

Reply:
Yeah, this would be a better approach, though even with the clobber evaluation, we might still make extra reg copies if v3 is commoned.  If VFMA is non destructive and same performacne as MADBR/MEDBR, we might want to prefer that on supported hardware levels.